operations/services.md

```diff
@@ -90,10 +121,24 @@ Returns all services offered by the enterprise.
 | `AfterCheckOut` | boolean | required | Whether it can be promoted after check-out. |
 | `DuringCheckOut` | boolean | required | Whether it can be promoted during check-out. | 
 
-#### Service type
+### Offsets and time units details
```

Comment:
@vitasamek @siroky could you please check if this kind of explanation is understandable and enough? I would let the content team to polish the wording there and probably polish the colors used in the diagram, but I do not want to waste more time atm so rather asking before I do, if you think the representation is good or you would choose some different approach, but I think that some kind of diagram will be needed to properly understand the offsets in the connector API.

Reply:
It makes sense to me! I liked it. With the image is pretty clear.

Reply:
@siroky @vitasamek thanks for the feedback 🚀 , I asked Kyle to have a look at the wording, I will update when done and move from draft to ready for review